pkg/render/logstorage.go

```diff
@@ -462,6 +467,48 @@ func (es *elasticsearchComponent) Ready() bool {
 	return true
 }
 
+// In managed clusters, we need to provision a role and binding for linseed to provide permissions
+// to create configmaps.
+func (es elasticsearchComponent) linseedExternalRoleAndBinding() (*rbacv1.ClusterRole, *rbacv1.RoleBinding) {
+	// Create a ClusterRole to provide configmap permissions. However, we'll only bind this to
+	// specific namespaces using RoleBindings so that we only have permissions in our namespaces.
+	role := &rbacv1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "tigera-linseed",
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{""},
+				Resources: []string{"secrets"},
+				Verbs:     []string{"create", "update", "get", "list"},
```

Comment:
Will we need delete in the future? We would probably need to think about clean -up when a managed cluster is disconnected.